src/PKSim.Core/Services/IndividualSimulationEngine.cs

```diff
@@ -65,10 +62,10 @@ private void terminated()
             _eventPublisher.PublishEvent(new SimulationRunStartedEvent());
             await runSimulation(individualSimulation, exportAll: false, raiseEvents: true, checkForNegativeValues: true);
          }
-         catch (Exception ex)
+         catch (Exception)
          {
-            _exceptionManager.LogException(ex);
```

Comment:
This was the wrong place to catch the exception. Should be done higher in the chain